### DISCUSSION TITLE
断剑5 汉化版

### DISCUSSION THREAD

**Discussion by ghost** [Gamuxorg/bbs#363] [投递和请求游戏]

**Comment by ghost** [comment]
有没有大神能搞到？

**Comment by shouhuanxiaoji** [comment]
游戏有点大啊。。周末的吧

**Comment by ghost** [comment]
@shouhuanxiaoji 你解决汉化了吗？

**Comment by shouhuanxiaoji** [comment]
![Image of Yaktocat](https://img.vim-cn.com/f4/06d6165b47a5f790ffd0377855c0bc4e2f41ea.png)

**Comment by ghost** [comment]
@shouhuanxiaoji 你是怎么解决的我打补丁后就启动不了游戏显示缺少一个英文字体,游戏是steam下的。汉化补丁是这下的https://steamcn.com/t202760-1-1 就是你用的这个啊。为什么我用不了

**Comment by shouhuanxiaoji** [comment]
我的游戏不是steam下的，大概这是区别？你需要吗，我可以上传一份

**Comment by ghost** [comment]
@shouhuanxiaoji 需要啊

**Comment by shouhuanxiaoji** [comment]
https://dl1.linuxgame.cn/brokensword5/brokensword5_1.0_amd64.appimage

**Comment by ghost** [comment]
@shouhuanxiaoji 下载不了用uget也不行

**Comment by lq95v5** [comment]
@spphinslove 因为不小心被我删掉了 你等会 我找找。。

**Comment by ghost** [comment]
@lq95v5 官网还是没有啊

**Comment by shouhuanxiaoji** [comment]
要重新打包上传

**Comment by lq95v5** [comment]
@spphinslove  没打包 
你下载自己打汉化吧
http://pan.linuxgame.cn:8081/BrokenSword5.tar.xz
